posts/2021-the-bottom-line.md

```diff
@@ -0,0 +1,177 @@
+
+# New mathlib maintainers
+
+  The group of mathlib maintainers has also grown! In 2021, [Rémy Degenne](https://github.com/remydegenne), [Markus Himmel](https://github.com/TwoFX), [Bhavik Mehta](https://github.com/b-mehta), [Oliver Nash](https://github.com/ocfnash), [Adam Topaz](https://github.com/adamtopaz), and [Eric Wieser](https://github.com/eric-wieser) joined the team, which now numbers 23. (With the latter three, the team finally includes surnames from the second half of the alphabet.)
```

Comment:
```suggestion
The group of mathlib maintainers has also grown! In 2021, [Rémy Degenne](https://github.com/remydegenne), [Markus Himmel](https://github.com/TwoFX), [Bhavik Mehta](https://github.com/b-mehta), [Oliver Nash](https://github.com/ocfnash), [Adam Topaz](https://github.com/adamtopaz), and [Eric Wieser](https://github.com/eric-wieser) joined the team, which now numbers 23. (With the latter three, the team finally includes surnames from the second half of the alphabet.)
```

Reply:
I think it would be good to add a link to https://leanprover-community.github.io/meet.html#maintainers somewhere in this paragraph.